_migration-assistant/overview/is-migration-assistant-right-for-you.md

```diff
@@ -89,65 +77,91 @@ Before starting a migration, consider the scope of the components involved. The
+#### Deploying into a New VPC
 
-### Combination of all three
+- Migration Assistant provisions a new VPC with required components (e.g., NAT gateway, subnets).
+- You must establish network access from this VPC to both the source (or S3) and target clusters.
 
-Use a combination of all three migration types if:
+#### Deploying into an Existing VPC
 
-- You're performing a complex, multi-version migration.
-- You require zero downtime and full validation of the target environment.
-- You want end-to-end tooling for metadata, data movement, and cluster behavior comparison.
-- You're cloning an existing cluster and changing the source's configuration.
-- You're setting up disaster recovery.
+- If installing Migration Assistant into an existing VPC (e.g., same VPC as the source or target), you may need to configure connectivity to any cluster external to the target VPC.
+  - For example, if installed in the source VPC, you may need VPC endpoints or peering to reach the target.
+  
+- Ensure all required AWS services are reachable by Migration Assistant components.
 
-## Checklist
+  - If the VPC has outbound access using:
+    - Private subnets with a NAT gateway, or
+    - Public subnets with an Internet Gateway
 
-Use this checklist to decide whether Migration Assistant is right for you:
+    then VPC interface endpoints are not required.
 
-- Are you migrating across one or more major versions?
+  - If using isolated subnets with no outbound access, you must configure VPC interface endpoints or routing to the following services:
 
-- Do you need to maintain service availability with zero downtime?
+    - **Amazon Application Load Balancer (ALB)** – *(`Capture-and-Replay` only)* - Used to optionally reroute client traffic from the source to the target during migration.
+    - **Amazon CloudWatch** – Publishes migration metrics.
+    - **Amazon CloudWatch Logs** – Ingests ECS task logs.
+    - **Amazon EC2** – Used to bootstrap Migration Assistant. When deploying using CloudFormation, the bootstrap EC2 instance requires outbound internet access (using NAT gateway or IGW) to download the latest release from GitHub.
+    - **Amazon Elastic Block Store (Amazon EBS)** – Provides temporary disk storage.
+    - **Amazon Elastic Container Registry (Amazon ECR)** – Pulls container images.
+    - **Amazon Elastic Container Service (Amazon ECS)** – Orchestrates container workloads.
+    - **Amazon Elastic File System (Amazon EFS)** – Stores persistent logs.
+    - **Amazon Managed Streaming for Apache Kafka** – *(`Capture-and-Replay` only)* - Used as a durable storage to capture and replay HTTP live traffic.
+    - **Amazon S3** – Stores and retrieves snapshots and artifacts.
+    - **AWS Elastic Load Balancing (ELB)** – *(Capture-and-Replay only)* – Used by the Migration Console to connect to the Application Load Balancer (ALB).
+    - **AWS Secrets Manager** – Stores credentials securely.
```

Comment:
nit: We could say only needed when using basic auth for source or target cluster